packages/block-library/src/spacer/edit.js

```diff
@@ -24,6 +31,133 @@ const MAX_SPACER_HEIGHT = 500;
 const MIN_SPACER_WIDTH = 1;
 const MAX_SPACER_WIDTH = 500;
 
+function DimensionInput( {
```

Comment:
Would it be worth exporting these smaller components and adding some unit tests? Also, I know it is common to have multiple functional components in a single file, but I wonder it is worth trying to move more to a single file -> single component approach? I don't have a strong opinion on it, but it does some that some of the core block files can get quite long and complicated.